docs/cheatsheet/cheatsheet.html

```diff
@@ -0,0 +1,148 @@
+EarlGrey
+</div>
+</div>
+
+<div class="body">
+<ul>
+  <li>View Matchers</li>
+  <li>User Properties
+    <ul>
+      <li>grey_accessibilityID(...)</li>
+      <li>grey_accessibilityLabel(...)</li>
+      <li>grey_accessibilityValue(...)</li>
+      <li>grey_accessibilityTrait(...)</li>
+      <li>grey_accessibilityHint(...)</li>
+      <li>grey_text(...)</li>
+    </ul>
+  </li>
+  <li>UI Properties
+    <ul>
+      <li>grey_minimumVisiblePercent(...)</li>
+      <li>grey_sufficientlyVisible()</li>
+      <li>grey_interactable()</li>
+      <li>grey_notVisible()</li>
+      <li>grey_systemAlertViewShown()</li>
+      <li>grey_enabled()</li>
+      <li>grey_layout(...)</li>
+      <li>grey_nil()</li>
+      <li>grey_notNil()</li>
+    </ul>
+  </li>
+  <li>Hierarchy
+    <ul>
+      <li>grey_ancestor(...)</li>
+      <li>grey_descendant(...)</li>
+    </ul>
+  </li>
+  <li>Class
+    <ul>
+      <li>grey_kindOfClass(...)</li>
+      <li>grey_respondsToSelector(...)</li>
+      <li>grey_conformsToProtocol(...)</li>
+    </ul>
+  </li>
+  <li>Other
+    <ul>
+      <li>grey_keyWindow()</li>
+      <li>grey_firstResponder()</li>
+      <li>grey_accessibilityElement()</li>
+      <li>grey_progress(...)</li>
+      <li>grey_buttonTitle(...)</li>
+      <li>grey_stepperValue(...)</li>
+      <li>grey_sliderValueMatcher(...)</li>
+      <li>grey_pickerColumnSetToValue(...)</li>
+      <li>grey_datePickerValue(...)</li>
+      <li>grey_switchWithOnState(...)</li>
+      <li>grey_closeTo(...)</li>
+    </ul>
+  </li>
+  <li>Object Matchers
+  <ul>
+    <li>grey_allOfMatchers(...)</li>
+    <li>grey_anyOfMatchers(...)</li>
+  </ul>
+  </li>
+</ul>
+
+<ul>
+  <li>View Actions</li>
+  <li>Click/Press
+    <ul>
+      <li>grey_tap()</li>
+      <li>grey_tapAtPoint(...)</li>
+      <li>grey_doubleTap()</li>
+      <li>grey_longPress()</li>
+      <li>grey_longPressWithDuration(...)</li>
+      <li>grey_longPressAtPointWithDuration(...)</li>
+    </ul>
+  </li>
+  <li>Gestures
+    <ul>
+      <li>grey_scrollInDirection(...)</li>
+      <li>grey_scrollToContentEdge(...)</li>
+      <li>grey_swipeFastInDirection(...)</li>
+      <li>grey_swipeSlowInDirection(...)</li>
+      <li>grey_swipeFastInDirectionWithStartPoint(...)</li>
+      <li>grey_swipeSlowInDirectionWithStartPoint(...)</li>
+    </ul>
+  </li>
+  <li>Text
+    <ul>
+      <li>grey_typeText(...)</li>
+      <li>grey_clearText()</li>
+    </ul>
+  </li>
+  <li>Other
+    <ul>
+      <li>grey_moveSliderToValue(...)</li>
+      <li>grey_setStepperValue(...)</li>
+      <li>grey_turnSwitchOn(...)</li>
+      <li>grey_setDate(...)</li>
+      <li>grey_setPickerColumnToValue(...)</li>
+      <li>grey_javaScriptExecution(...)</li>
+      <li>grey_snapshot(...)</li>
+    </ul>
+  </li>
+</ul>
+
+<ul>
+  <li>Assertions</li>
+  <li>Grey Assert
+    <ul>
+      <li>GREYAssert(...)</li>
+      <li>GREYAssertTrue(...)</li>
+      <li>GREYAssertFalse(...)</li>
+      <li>GREYAssertNotNil(...)</li>
```

Comment:
Wasn't able to comment on the PDF so adding my comment here:
In the generated PDF, there seems to be a line break after this.